### PR TITLE
Use `PAGE_PARAMS` to specify page parameters

### PR DESCRIPTION
**Pull request by shivamMg**
- Provide description and type for page params
- Minor changes to documentation

https://github.com/fossasia/open-event-orga-server/issues/464 and https://github.com/fossasia/open-event-orga-server/issues/472

### CODE REVIEW

File: open_event/api/utils.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
 from flask_restplus import Model, fields, reqparse
 
+# Parameters for a paginated response
+PAGE_PARAMS = {
+    'start': {'description': 'ID to begin from', 'type': int},
```
Comment (aviaryan) [inline open_event/api/utils.py]: `start` is not the ID but the position. Say if event 3 has sessions with ids 3,5,7,9,10 then for `start=3` everything after session_id=7 in event 3 is elligible for return.

File: open_event/api/utils.py [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+    'limit': {'description': 'Number of results to be retrieved', 'type': int},
```
Comment (aviaryan) [inline open_event/api/utils.py]: Max number, max count..

**Comment by shivamMg** [comment]
@aviaryan Thanks for clearing that up. I guess the start parameter shows where relative ids would have been advantageous.

@juslee Please take a look. It just introduces a constant and minor changes to existing code.

**Comment by aviaryan** [comment]
@shivamMg hey, can you add the default values of start and limit (1 and 20 respectively) in the PAGE_PARAMS. 

@juslee To accelerate development, can we merge PRs ourselves if one or two other collaborators have verified the patch.

**Comment by shivamMg** [comment]
@aviaryan Nice catch. Done :+1:

**Comment by aviaryan** [comment]
@shivamMg please resolve the conflicts

**Pull request merged**